.github/workflows/linux.yml

```diff
@@ -77,17 +77,9 @@ jobs:
         restore-keys: |
           ${{ github.workflow }}-${{ matrix.python-version }}-ccache-linux-
 
-    - name: Setup build
+    - name: Setup build and install scipy
       run: |
-        meson setup build --prefix=$PWD/${{ env.INSTALLDIR }} --werror
-
-    - name: Build SciPy
-      run: |
-        ninja -C build -j 2
-
-    - name: Install SciPy
-      run: |
-        meson install -C build
+        python dev.py -rb --installdir ${{ env.INSTALLDIR }} -j 2 --build-only --werror
```

Comment:
This does look more concise. One thing I liked about the original is that the build and install steps are separate, so the ~1500 lines of output for the install step can be ignored easily when looking at build issues. Maybe that's something to keep - have to think about it more.